straxen/plugins/events/multi_scatter.py

```diff
@@ -3,47 +3,31 @@
 import numpy as np
 export, __all__ = strax.exporter()
 
+
 @export
 class EventInfoMS(strax.Plugin):
     """
-    Get the sum of s2 and s1,
-    Get the sum of cs2 inside the drift length. 
+    Plugin to collect multiple-scatter event observables
     """
-    __version__ = '0.0.1'
-    depends_on = ('event_info', 'peak_basics','peaks_per_event','peaks_corrections')
-    provides = 'event_MS_naive'
-
-    def infer_dtype(self):
-        dtype = []
-        dtype += [
-            ((f's1_sum'), np.float32),
-            ((f'cs1_multi'), np.float32),
-            ((f'cs1_multi_wo_timecorr'), np.float32),
-            ((f's2_sum'), np.float32),
-            ((f'cs2_sum'), np.float32),
-            ((f'cs2_wo_timecorr_sum'), np.float32),
-            ((f'cs2_wo_elifecorr_sum'), np.float32),
-            ((f'cs2_area_fraction_sum'), np.float32),
-            ((f'ces_sum'), np.float32),
-            ((f'e_charge_sum'), np.float32),
-             ]
-        dtype += strax.time_fields
-        return dtype
-
+    __version__ = '0.0.2'
+    depends_on = (
+        'event_info',
+        'peak_basics', 'peak_per_event', 'peak_corrections', 'peak_positions')
+    provides = 'event_ms_naive'
```

Comment:
Do you mind if I change this to `event_ms_naive`? It is better to use lowercase in `provides`.